components/tidb_query/src/codec/mysql/json/json_modify.rs

```diff
@@ -28,11 +28,11 @@ impl Json {
         mt: ModifyType,
     ) -> Result<()> {
         if path_expr_list.len() != values.len() {
-            return Err(box_err!("Incorrect parameter count"));
+            return Err(box_err!("Incorrect parameter: {:?}", path_expr_list));
         }
         for expr in path_expr_list {
             if expr.contains_any_asterisk() {
-                return Err(box_err!("Invalid path expression"));
+                return Err(box_err!("Invalid path expression: {:?}", path_expr_list));
```

Comment:
Could you use the specific expression where there is an error, rather than all of the expressions? You could also add some detail about why it is invalid.

Reply:
Fixed